rpc/ethereum/backend/utils.go

```diff
@@ -155,5 +155,11 @@ func TxLogsFromEvents(events []abci.Event) ([]*ethtypes.Log, error) {
 			logs = append(logs, &log)
 		}
 	}
-	return evmtypes.LogsToEthereum(logs), nil
+	ethLogs := evmtypes.LogsToEthereum(logs)
```

Comment:
nitpicking but maybe you can also improve performance of `LogsToEthereum` 

by initializing 

```
ethLogs := make([]common.Hash, len(logs))
...

ethLogs[i] = logs[i].ToEthereum()